app/src/renderer/vuex/modules/delegates.js

```diff
@@ -41,13 +39,15 @@ export default ({ dispatch, node }) => {
     async getDelegates({ state, dispatch, commit, rootState }) {
       commit("setDelegateLoading", true)
       let delegates = await node.candidates()
+      let { validators } = await node.getValidators()
```

Comment:
this is only happening once. can we use the rpc event to requery the validators when they change?

Reply:
Put this as a follow up https://github.com/cosmos/voyager/issues/946